packages/utils/src/formatDate.ts

```diff
@@ -0,0 +1,70 @@
+import { differenceInDays, formatDistanceToNowStrict } from "date-fns"
+import { nb, enUS } from "date-fns/locale"
+
+const DEFAULT_DAYS_RELATIVE_THRESHOLD = 3
+const DEFAULT_LOCALE = "nb-NO"
+
+const dateFNSLocaleFromIntlLocale = {
+  "nb-NO": nb,
+  "en-US": enUS,
+} as const
+
+type SupportedLocales = keyof typeof dateFNSLocaleFromIntlLocale
+
+/**
+ * Formats a date into a string representation. If the date is within the date threshold, the date will be formatted as a relative date.
+ *
+ * @param date - The date to format.
+ * @param options - Optional formatting options.
+ * @param options.forceAbsolute - If set to true, the date will always be formatted using the `formatRemainingTime` function.
```

Comment:
Is the description wrong here? `forceAbsolute` makes me think it should _never_ be relative, instead of _always_ relative? Also I think a better naming would be `neverRelative`